docs/api/saved-objects/bulk_create.asciidoc

```diff
@@ -41,7 +41,7 @@ experimental[] Create multiple {kib} saved objects.
 `references`::
   (Optional, array) Objects with `name`, `id`, and `type` properties that describe the other saved objects in the referenced object. To refer to the other saved object, use `name` in the attributes. Never use `id` to refer to the other saved object. `id` can be automatically updated during migrations, import, or export.
 
-`namespaces`::
+`initialNamespaces`::
```

Comment:
Maybe it's worth clarifying in the description that if this is provided, the current space is ignored. Can be done in follow up.

Reply:
Submitted a follow-up PR: #79769